maps/map_files/DesertDam/Desert_Dam.dmm

```diff
@@ -65095,9 +65095,6 @@
 	dir = 8
```

Comment:
Trijent Dam:
- Noop turfs and areas when there are no changes between your insert in certain locations and the base map itself.
- ![bilde](https://user-images.githubusercontent.com/59782240/184449471-429cc6a4-2217-4c63-bb4f-3ea3aaa5c534.png)
- ![bilde](https://user-images.githubusercontent.com/59782240/184449618-b98018fa-b6db-475f-97bb-27e2aeaf17c5.png)
- What happened to all the weapons and armory? Did the CLF troopers just not use them? It looks to clean in this armor, also, the walls are too strong around the armory. Drop the guns around the area on the corpses or nearby, and limit the manually spawned guns if these corpses spawn with guns. Also remember CLF are in general underfunded and lack good equipment.
- ![bilde](https://user-images.githubusercontent.com/59782240/184449835-b561ef00-541b-428a-b5d2-ec0cc3075229.png)
- Change the Health on the supposed damaged barricades or they'll just look damaged till they take dmg and change their sprite to be fine again.
- ![bilde](https://user-images.githubusercontent.com/59782240/184449994-e9fe6d37-f12c-4fc5-bd56-5e3f7185f9d1.png)
- Chill on the CLF posters, a few here and there is fine, don't need as many as you currently have.